tutorials/math/interpolation.rst

```diff
@@ -138,4 +142,48 @@ Here is how it looks:
 
 .. image:: img/interpolation_follow.gif
 
-This is useful for smoothing camera movement, allies following you (ensuring they stay within a certain range), and many other common game patterns.
+This is useful for smoothing camera movement, for allies following the player
+(ensuring they stay within a certain range), and for many other common game patterns.
+
+.. note::
```

Comment:
I'd say this whole section in the note needs to be further indented or it's nearly impossible to tell what is in the note or not, I'd say to make it at least 4 spaces

Reply:
Agree, it was bothering me as I wrote it. The whole page might need to change, I'm not sure if RST/Sphinx handles pages with mixed spaces per indentation well.

Reply:
Ah, it looks like I was doing something wrong last time I tried using mixed spaces per indentation in a single page. It compiles fine for me now.

Just to be clear, your recommendation is 4 spaces per indentation like this?
```
.. note::
    Paragraph of text.

    .. tabs::    
        .. code-tab:: gdscript GDScript

            code

        .. code-tab:: csharp

            code {}

    Paragraph of text.
```

Not 2 spaces per indentation, like this?
```
.. note::
  Paragraph of text.

  .. tabs::    
    .. code-tab:: gdscript GDScript

        code

    .. code-tab:: csharp

        code {}

  Paragraph of text.

```

Reply:
I'd say either goes, 3 might be good too to line up with the "note" text, but it should be more than 1

Reply:
It's definitely spaces, not tabs, in the manual.

3 seems wrong. Most pages use either 2 or 4. This page is an exception in using 1.

As a rule I prefer 4 spaces per indentation because it makes the page more legible, but I believe 2 may be chosen in some cases because it minimizes total line length?

Reply:
I'd say 4 then, to make this unusually long note less confusing